### PR TITLE
Feat/radiobutton

### PR DESCRIPTION
**Pull request by sergdeev**

### CODE REVIEW

File: src/components/radiobutton/index.tsx [hunk 1]
```diff
@@ -50,7 +65,7 @@ export const Radio = React.forwardRef<HTMLLabelElement, RadioButtonProps>(
             className={cx('radiobox', inputClassName)}
             id={id || defaultId}
             disabled={disabled}
-            checked={checked || value === id}
+            checked={checked || defaultId === id}
```
Comment (vozbrann) [inline src/components/radiobutton/index.tsx:68]: breaking changes

**Comment by vozbrann** [comment]
It looks different from a prev version 
Old:
image

New:
image

File: src/components/radiobutton/index.tsx [hunk 2]
```diff
@@ -40,8 +54,8 @@ export const Radio = React.forwardRef<HTMLLabelElement, RadioButtonProps>(
   ): JSX.Element => {
     const theme = useTheme();
     const uniqueID = idGenerator();
-    const defaultId = id || uniqueID;
-    const { className: inputClassName, ...otherInputProps } = inputProps;
+    const defaultId = uniqueID;
+    const { className: inputClassName, id, ...otherInputProps } = inputProps;
```
Comment (vlacher12) [inline src/components/radiobutton/index.tsx:58]: do we still need default id?

**Review by vlacher12 — CHANGES_REQUESTED** [review]

File: src/components/radiobutton/index.tsx (hunk 2 above)
Comment (vozbrann) [reply src/components/radiobutton/index.tsx]: fixed

**Review by vlacher12 — APPROVED** [review]

**Review by lazzystep — APPROVED** [review]

**Pull request merged**